plugins/node/opentelemetry-instrumentation-express/package.json

```diff
@@ -0,0 +1,70 @@
+{
+  "name": "@opentelemetry/instrumentation-express",
+  "version": "0.13.1",
+  "description": "OpenTelemetry express automatic instrumentation package.",
+  "main": "build/src/index.js",
+  "types": "build/src/index.d.ts",
+  "repository": "open-telemetry/opentelemetry-js-contrib",
+  "scripts": {
+    "test": "nyc ts-mocha -p tsconfig.json 'test/**/*.ts'",
+    "codecov": "nyc report --reporter=json && codecov -f coverage/*.json -p ../../",
+    "tdd": "yarn test -- --watch-extensions ts --watch",
+    "clean": "rimraf build/*",
+    "lint": "eslint . --ext .ts",
+    "lint:fix": "eslint . --ext .ts --fix",
+    "precompile": "tsc --version",
+    "version:update": "node ../../../scripts/version-update.js",
+    "compile": "npm run version:update && tsc -p .",
+    "prepare": "npm run compile",
+    "watch": "tsc -w"
+  },
+  "keywords": [
+    "opentelemetry",
+    "express",
+    "nodejs",
+    "tracing",
+    "profiling",
+    "instrumentation"
+  ],
+  "author": "OpenTelemetry Authors",
+  "license": "Apache-2.0",
+  "engines": {
+    "node": ">=8.5.0"
+  },
+  "files": [
+    "build/src/**/*.js",
+    "build/src/**/*.d.ts",
+    "doc",
+    "LICENSE",
+    "README.md"
+  ],
+  "publishConfig": {
+    "access": "public"
+  },
+  "devDependencies": {
+    "@opentelemetry/context-async-hooks": "0.17.0",
+    "@opentelemetry/node": "^0.17.0",
+    "@opentelemetry/tracing": "^0.17.0",
```

Comment:
```suggestion
    "@opentelemetry/tracing": "0.17.0",
```

Reply:
Why do we pin this version specially ?

Reply:
As far as I understood all dev-dependencies are pinned. If you don't do it some bot creates a PR to do it.

maybe @dyladan knows the background.

Reply:
This is the default behavior of the renovate bot we use. The reason to pin development dependencies is so you can be sure that every developer is using the same version which prevents things like my machine showing a linting error that another machine does not. The runtime dependencies are ranged with `^` so that we can be sure our customers receive any important security patches without us having to publish a new release.

Reply:
I'll pin the deps tomorrow

Reply:
shouldn't this be `0.15`, and if this is merged then it should be changed to 0.17 before release (0.16 will be an orphaned anyway)

Reply:
We cant wait for https://github.com/open-telemetry/opentelemetry-js-contrib/pull/337, i think that make more sense

Reply:
the thing is that if you merge this, and then use a github directly instead of npm, this instrumentation will depend on different version then other instrumentation / plugins in repo. This will cause that they will be using 2 different version of api so you won't be able to use the plugins and instrumentation together even though they have the same version. So  that's why it is important to keep version aligned and change dependency once for all.

Reply:
I don't really don't get whats you are trying to explain. `0.17.0` is the latest version currently so i don't see why we shouldn't make instrumentation depend on it.

Reply:
if this instrumentation depends on 0.17 and all the rest packages depends on 0.15 you won't be able to even test them together before they are released. Try to simply use now express with any other package from contrib. In mono repo all packages should depend on the same version of the core. This change 0.17 should be done once for all packages.

Reply:
They can't be used together since you'll need to choose an api version (either 0.15 for current plugins or 0.17 for this) but we will not publish them to npm so i don't see how thats a problem.

Anyway we can just wait for contrib to be updated to 0.17 which i believe daniel is working on before merging it.